plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueHiveMetastore.java

```diff
@@ -1126,4 +1130,29 @@ public void revokeTablePrivileges(String databaseName, String tableName, String
     {
         return ImmutableSet.of();
     }
+
+    static class StatsRecordingAsyncHandler<Request extends AmazonWebServiceRequest, Result>
```

Comment:
Seems like this belongs in `GlueMetastoreApiStats`

Reply:
Let's change it as a followup :)